package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "esbuild-plugin-preserve-directives",
-  "version": "0.0.8",
+  "version": "0.0.9",
```

Comment:
Hello, everyone!
This part will be automatically changed through 'changeset', so I think it can be removed from this PR!